src/main/java/org/folio/rest/impl/OaiPmhImpl.java

```diff
@@ -66,8 +66,14 @@
 
   /** Map containing OAI-PMH verb and corresponding helper instance. */
   private static final Map<VerbType, VerbHelper> HELPERS = new EnumMap<>(VerbType.class);
+  private static final int INSTANCES_EXPIRATION_TIME_IN_SECONDS = 7200;
```

Comment:
What is the API behaviour if the request with the resumption token is invoked after 2 hours?

Reply:
The description mentions 1 day but here it just has 2 hours?

Reply:
Thank you for good note. Changed to 86400 seconds. Regarding your first question I've explained it above.